boards/arm/nrf52840_papyr/board.cmake

```diff
@@ -1,3 +1,5 @@
 # SPDX-License-Identifier: Apache-2.0
 
+board_runner_args(nrfjprog "--nrf-family=NRF52" "--softreset")
```

Comment:
Doesn't it make it default now?

Reply:
No, the order of the includes specifies the default.